bundles/net.tourbook/src/net/tourbook/ui/views/tourBook/TourBookView.java

```diff
@@ -962,10 +959,13 @@ private void defineAllColumns(final Composite parent) {
       defineColumn_Altitude_Up();
       defineColumn_Altitude_Down();
       defineColumn_Altitude_Max();
+      defineColumn_Avg_Altitude_Change();
```

Comment:
All altitude columns should start with `defineColumn_Altitude`

My suggestion is `defineColumn_Altitude_AvgChange`

Reply:
Done (see PR)